src/chains/serializers.py

```diff
@@ -111,18 +117,25 @@ class BlockExplorerUriTemplateSerializer(serializers.Serializer[Chain]):
     api = serializers.URLField(source="block_explorer_uri_api_template")
 
 
-class FeatureSerializer(serializers.Serializer[Feature]):
+class FeatureSerializer(serializers.ModelSerializer[Feature]):
     class Meta:
+        fields = "__all__"
+        model = Feature
         ref_name = "chains.serializers.FeatureSerializer"
 
     @staticmethod
-    def to_representation(instance: Feature) -> str:
+    def to_representation(instance: Feature) -> str:  # type: ignore[override]
         return instance.key
 
 
-class WalletSerializer(serializers.Serializer[Wallet]):
+class WalletSerializer(serializers.ModelSerializer[Wallet]):
+    class Meta:
+        fields = "__all__"
```

Comment:
Similar to the above:

```python
fields = ['key']
```

Reply:
Fixed in 4b2a288431b8ec3ab414b4c7f2e7a99eedc8e547.